src/containers/GrantsTableWrapper.js

```diff
@@ -0,0 +1,214 @@
+import React from 'react';
+
+import { uniq, map, filter, findIndex, sortBy } from 'lodash';
+
+import { useQuery, gql } from '@apollo/client';
+
+import { stripHtml, extractYear } from '../utils';
+
+import GrantTable from '../components/GrantTable';
+
+const GET_ORGANIZATION_GRANTS = gql`
+  query getOrg($organizationId: String!, $grantsFundedOffset: Int, $grantsReceivedOffset: Int, $limit: Int) {
+    organization(uuid: $organizationId) {
+      grantsFunded (offset: $grantsFundedOffset, limit: $limit, orderBy: uuid) {
+        uuid
+        dateFrom
+        dateTo
+        to {
+          name
+          uuid
+        }
+        amount
+        description
+      }
+      grantsReceived (offset: $grantsReceivedOffset, limit: $limit, orderBy: uuid) {
+        uuid
+        dateFrom
+        dateTo
+        from {
+          name
+          uuid
+        }
+        amount
+        description
+      }
+    }
+  }
+`;
+
+const GrantsTableWrapper = (props) => {
+  const { loading, error, data, fetchMore } = useQuery(GET_ORGANIZATION_GRANTS, {
+    variables: { 
+      organizationId: props.organizationId,
+      grantsFundedOffset: 0,
+      grantsReceivedOffset: 0,
+      limit: 100
+    },
+    notifyOnNetworkStatusChange: true,  // Update 'loading' prop after fetchMore
+  });
+
+  if (loading) return 'Loading...';
+  if (error) return `Error! ${error}`;
+
+  if (!data.organization) return `Failed to load org data!`;
+
+  if (data.organization.grantsFunded.length < props.countGrantsFrom || data.organization.grantsReceived.length < props.countGrantsTo) {
+    fetchMore({
+      variables: {
+        grantsFundedOffset: data.organization.grantsFunded.length,
+        grantsReceivedOffset: data.organization.grantsReceived.length,
+      },
+    });
+  }
+
+  console.log(
+    `Fetched ${data.organization.grantsFunded.length}/${props.countGrantsFrom} grants funded\
+     and ${data.organization.grantsReceived.length}/${props.countGrantsTo} grants received.`
+  );
+
+  const {
+    grantsFunded,
+    grantsReceived,
+    fundedYearlySums,
+    receivedYearlySums,
+  } = cleanse(data.organization);
+
+  return (
+    <GrantTable
+      verb={props.showGrantSide}
+      grants={props.showGrantSide === 'funded' ? grantsFunded : grantsReceived}
+      sums={
+        props.showGrantSide === 'funded' ? fundedYearlySums : receivedYearlySums
+      }
+    />
+  );
+};
+
+const cleanse = (organization) => {
+  // Sort lists by org
+  const flattenedGrantsReceived = sortBy(
```

Comment:
This is a total personal preference, but I would advocate splitting this into two steps for clarity. First the `map` to generate the list, then a second step for the `sortBy`. 

The reason I suggest that is the `sortBy` call is ~20 lines away from the sort logic, so I was originally confused as to what was happening here. I thought it was super complicated sort logic! Then I realized it's lodash and not `Array.prototype.sort()`